sections/featured-product.liquid

```diff
@@ -229,6 +236,7 @@
                           {{ variant.title }}
                           {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
                           - {{ variant.price | money | strip_html }}
+                          {% if variant.available and variant.inventory_management == 'shopify' -%} - {{ 'products.product.inventory' | t: quantity: variant.inventory_quantity }}{% endif %}
```

Comment:
Maybe another check here to see if the block has been added or not ? So it's not there by default. 

It adds a bit of logic but follows the setting from the admin (enabled or not).

Reply:
Thanks Ludo. I chose to not add a check here for the `block.type` since there weren't any for the [title or price](https://screenshot.click/11-58-cra0e-h79i2-_-.png) and both of these are still [displayed](https://screenshot.click/11-59-s923z-36103-_-.png).

I'm interested to hear your thoughts as I'm sure there are other benefits to adding a check here. I have to reservations against this either so happy to add a check for the inventory block 👍🏼

Reply:
Yeah very good point. And it's such a small use case as well I think that it should be fine. 
I wonder if there are cases where a merchant wouldn't want to display the inventory quantity. As it would make it available in the source code as well. We can see what another reviewer might think once you get a second review 🙂